frontend/src/component/changeRequest/ChangeRequestOverview/ChangeRequestTimeline/ChangeRequestTimeline.tsx

```diff
@@ -102,17 +109,24 @@ export const ChangeRequestTimeline: FC<ISuggestChangeTimelineProps> = ({
             data = steps;
     }
     const activeIndex = data.findIndex((item) => item === state);
-    const { locationSettings } = useLocationSettings();
 
     return (
         <StyledPaper elevation={0}>
             <StyledBox>
                 <StyledTimeline>
                     {data.map((title, index) => {
+                        const timestampComponent =
+                            index <= activeIndex && timestamps?.[title] ? (
+                                <Timestamp timestamp={timestamps?.[title]} />
+                            ) : undefined;
```

Comment:
This is a stopgap solution: only show timestamps up to and including the current state. In other words, don't show timestamps for states after the current. 

The prime example as to why we want this is where you make changes to a CR after getting it approved. That will reset the CR to in review, but we'll still have a timestamp for when you entered the approved state. 

This is probably something we want to filter out on the back end, but for now, this takes care of it.